packages/core/src/FrameLoop.ts

```diff
@@ -197,8 +197,8 @@ export class FrameLoop {
         const numSteps = Math.ceil(dt / step)
 
         for (let n = 0; n < numSteps; ++n) {
-          const springForce = (-config.tension! / 1000000) * (position - to)
-          const dampingForce = (-config.friction! / 1000) * velocity
+          const springForce = -config.tension! * 0.000001 * (position - to)
+          const dampingForce = -config.friction! * 0.001 * velocity
```

Comment:
Is this faster?

Reply:
By a infinitesimal fraction (I've seen this elsewhere), but it mostly helps getting rid of the parenthesis :)